### PR TITLE
Create environment.yml

### PR DESCRIPTION
**Pull request by marcin-mulawa**

### CODE REVIEW

File: environment.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+name: ml-cdv
+channels:
+ - conda-forge
+ - defaults
+dependencies:
+ - python== 3.8.2
```
Comment (mkcdv) [inline environment.yml]: Please remove whitespace after `==`

File: environment.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+ - pandas
+ - pytest
+ - isort
+ - pre-commit
+ - flake8
+ - pip
```
Comment (mkcdv) [inline environment.yml]: The `pip` is not used.  Please remove or add any package

**Review by mkcdv — CHANGES_REQUESTED** [review]
Everything works fine just polish sth

File: environment.yml (hunk 1 above)
Comment (marcin-mulawa) [reply environment.yml]: done

File: environment.yml (hunk 2 above)
Comment (marcin-mulawa) [reply environment.yml]: fixed

**Pull request merged**